partiql-parser/src/test/kotlin/org/partiql/parser/internal/PartiQLParserDDLTests.kt

```diff
@@ -77,27 +107,232 @@ class PartiQLParserDDLTests {
                     null
                 )
             ),
+
+            //
```

Comment:
nice set of tests. could also consider adding a few more:
- multiple constraints
- search condition w/ an `OR` (like what was mentioned in the ANTLR grammar
- other parsing error edge cases
  - missing parens
  - table constraint used where attribute constraint should be placed (and vice-versa)

Reply:
Good suggestion, we can add more tests once the parser portion of DDL implementation is completed. Since we are going to add more features, the modeling will change hence the test code.